src/relayFeeCalculator/chain-queries/zksync.ts

```diff
@@ -26,3 +26,27 @@ export class ZkSyncQueries extends QueryBase {
     );
   }
 }
+
+export class zkSyncGoerliQueries extends QueryBase {
```

Comment:
Should we comment on this class/file?

Reply:
Done: https://github.com/across-protocol/sdk-v2/pull/367/commits/dcde52c57286e8fdedc8e52bf7a806c99c687a12